src/gsi/gsimod.F90

```diff
@@ -1437,6 +1440,8 @@ subroutine gsimain_initialize
        if(mype == 0) write(6,*) ' ltlint = true, so vqc and njqc must be false'
      end if
   end if
+  if (fv3sar_bg_opt.ne.0) l_reg_update_hydro_delz=.false.
+  write(6,*)'thinkdeb333 bg_opt and *o_dz is ',fv3sar_bg_opt, l_reg_update_hydro_delz
```

Comment:
What is the importance of thinkdeb333?  If it is not important, please remove.

Reply:
removed this line